acoustid/future/worker.py

```
@@ -0,0 +1,79 @@
+import asyncio
+import logging
+from contextlib import AsyncExitStack
+from typing import Iterable, List
```

Ruff (0.8.2)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)